### ISSUE TITLE
 [WARN] Line 89: [unusable-by-js]

### ISSUE THREAD

**Issue by bpetridean** [ascendtech/gwt-highcharts#2]
```
Warnings in us/ascendtech/highcharts/client/chartoptions/Data.java
[INFO]       [WARN] Line 89: [unusable-by-js] Return type of 'Integer Data.ParseDate.getParsedDate(String)' is not usable by but exposed to JavaScript.
[INFO]    [WARN] Suppress "[unusable-by-js]" warnings by adding a `@SuppressWarnings("unusable-by-js")` annotation to the corresponding member.
```

Changing return type to `int` should make the warning go away

**Comment by payammeyer** [comment]
Thanks for bringing this up. Through the years, I've learned that `Double` is a better choice for `number` types in JsInterop, so I've updated all references to `Integer` to `Double`. I've released version 1.2.0.

**Comment by urica** [comment]
@payammeyer  when this release will be available in repo? https://maven.ascend-tech.us/repo/us/ascendtech/highcharts/

**Comment by payammeyer** [comment]
@urica My apologies, I didn't check to see if it actually published, looks like I had many fixes to make after the gradle version bump, 1.2.0 is now available on the repo. Please let me know if there are any issues.
